QuizApp/QuizApp/UI/QuizDetails/QuizDetailsViewController.swift

```diff
@@ -0,0 +1,143 @@
+import Combine
+import UIKit
+
+class QuizDetailsViewController: UIViewController {
+
+    private let gradient = CAGradientLayer()
+
+    private var viewModel: QuizDetailsViewModel!
+    private var titleLabel: UILabel!
+    private var backButtonImage: UIImage!
+    private var leaderboardButton: UIButton!
+    private var quizDetailsView: QuizDetailsView!
+    private var cancellables = Set<AnyCancellable>()
+    private var quiz: Quiz = Quiz(
```

Comment:
Why do you need to store quiz in VC? If it is so that you can pass ID on Start, remember that VM already has a reference to quiz ;)